code/src/components/app-table/components/app-table-sort-label/AppTableSortLabel.types.ts

```diff
@@ -0,0 +1,10 @@
+import { ReactNode } from "react";
+
+import { TableSortLabelProps } from "@mui/material/TableSortLabel";
+
+
+export type AppTableSortLabelProps = TableSortLabelProps & {
```

Comment:
Omit `active`, `direction` and `onClick` props because you already defained values for them

Reply:
Also fixed. Look one more time, please

Reply:
I meant like follows:

```tsx
export type AppTableSortLabelProps = Omit<
  TableSortLabelProps,
  "direction"
> & {
  sortDirection: SortOrder;
};
```

Reply:
Fixed